config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -123,6 +123,8 @@ presubmits:
     optional: true # remove this once the job is green
     branches:
     - main
+    - release-3.5
+    - release-3.4
```

Comment:
It seems that 3.5 and 3.4 do not have `run-govulncheck` in `Makefile`?

Reply:
cc @ivanvc

Reply:
> It seems that 3.5 and 3.4 do not have `run-govulncheck` in `Makefile`?

Yes, it needs to be backported

Reply:
Since there is no `test.sh` we can add the `govuln_pass()` function in `test_lib.sh` for 3.4, 3.5 branches.

Reply:
OR
we can fix the presubmit to not depend on Makefile and pass the direct commands to run

Reply:
> Since there is no `test.sh` we can add the `govuln_pass()` function in `test_lib.sh` for 3.4, 3.5 branches.

For `release-3.5` `test.sh` is at the root project directory not in `scripts/`.  Imo we should be backporting the `Makefile` recipe along with changes to `test.sh` so tests are run in a uniform way where possible. Otherwise we create cognitive burden when switching between working on different branches.